carpooling/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from . import controllers
+from . import models
```

Comment:
EOF :-p ( To avoid two lines on the diff views it's always better to add a new line on each files )